### DISCUSSION TITLE
jbang in hacktober fest

### DISCUSSION THREAD

**Discussion by maxandersen** [jbangdev/jbang#372] [General]
In case anyone is looking for a set of issues to hack on in October for hacktober fest I've added a few at https://github.com/jbangdev/jbang/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A"good+first+issue

They range from simple ones to some which are cool and nice features which I *think* should be fairly straight forward to add.

Ask away in this thread or on the issues and I/we will help!

Enjoy!

**Comment by gastaldi** [comment]
The link is broken. The correct is: 

https://github.com/jbangdev/jbang/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22

**Reply by maxandersen** [reply]
thanks! link fixed!
